CONTRIBUTING.md

```diff
@@ -189,6 +190,101 @@ library maintainers on the exclusion from this rule. In such cases there is a
 requirement to test unsafe code with sanitizers including Miri.
 
 
+### Policy
+
+We have various `rust-bitcoin` specific coding styles and conventions that are
+grouped here loosely under the term 'policy'. These are things we try to adhere
+to but that you should not need to worry too much about if you are a new
+contributor. Think of this as a place to collect group knowledge that exists in
+the various PRs over the last few years.
+
+
+#### Errors
+
+Return as much context as possible with errors e.g., if an error was encountered parsing a string
+include the string in the returned error type. If a function consumes costly-to-compute input
```

Comment:
This one I'm no longer sure about. Didn't have time to investigate deeply yet.

Reply:
Flagging, I have not changed these lines as yet.

Reply:
I now again believe it's the right thing to do